docs/technical_reference/unit_models/dewatering.rst

```diff
@@ -0,0 +1,75 @@
+Dewatering Unit
+===============
+
+.. index::
+   pair: watertap.unit_models.dewatering;dewatering
+
+The main assumptions of the implemented model are as follows:
+
+1) Single liquid phase only
+2) Steady state only
+3) Has no volume
+
+Introduction
+------------
+Dewatering is a process commonly used in wastewater treatment plants to separate water from solids using centrifugal motion
+and/or vacuums. In doing so, the dewatering unit reduces the sludge volume, which improves system operation and mitigates costs
+related to storage, processing, disposal, etc. This implementation of the dewatering unit is based on the `IDAES separator unit <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/separator.html>`_.
+
+Degrees of Freedom
+------------------
+The degrees of freedom in a dewatering unit model are the inlet feed state variables:
+
+    * temperature
+    * pressure
+    * component mass compositions
+
+Model Structure
+---------------
+The dewatering unit model does not use ControlVolumes, and instead writes a set of material, energy and momentum
+balances to split the inlet stream into two outlet streams. Dewatering models have a single inlet Port
+(named inlet) and two outlet Ports (named overflow and underflow).
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Particulate Components", ":math:`j`", "['X_I', 'X_S', 'X_P', 'X_BH', 'X_BA', 'X_ND']"
+   "Non-particulate Components", ":math:`j`", "['H2O', 'S_I', 'S_S', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'S_ALK']"
```

Comment:
I think it'd be nice to refer the reader to ASM for people to follow the component symbols